File: src/main/java/destiny/utils/I18nSupport.java
/**
 * @author smallufo
 * Created on 2010/10/10 at 下午9:32:54
 */
package destiny.utils;

import java.util.Locale;

public interface I18nSupport
{
  /** 取得資料庫中(或其他來源) 支援的 Locales */
  Iterable<Locale> getSupportedLocales();
}

File: src/main/kotlin/destiny/tools/I18nSupport.kt
/**
 * @author smallufo
 * Created on 2010/10/10 at 下午9:32:54
 */
package destiny.tools

import java.util.*

interface I18nSupport {

  /** 取得資料庫中(或其他來源) 支援的 Locales  */
  val supportedLocales: Iterable<Locale>
}
